[IMP] point_of_sale: Adyen Event URLs configuration (#3009)
odoo/documentation: content/applications/sales/point_of_sale/payment/adyen.rst

--> Terminals`, select the terminal to link, and copy-paste its serial number onto the Odoo
mandatory field.

Set the Event URLs

> ~~~~~~~~~~~~~~~~~~

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~~~

```
(there must be a blank line under titles

In order for Odoo to know when a payment have been made, the terminal `Event URLs` needs to be set.

> 
> To do this, please follow this steps:
> 
>  1. Connect to your **Adyen account**

Reviewer: Numbered lists should be written using the #. instead of actual numbers (in case the list has to change afterward)

```suggestion
#. log in on `Adyen's website <https://www.adyen.com/>`_;
```

 2. Once connected, go to :menuselection:`Point of Sale --> Terminals`
 3. Select the terminal that is linked

>  4. On the terminal setting, choose the option called `Integrations`

Reviewer: ```suggestion
 #. from the terminal settings, click :guilabel:`Integrations`;
```

 5. Next to `Switch to decrypted mode to edit this setting`, make sure that it is set to `Decrypted`

>  6. | Change `Event URLs` field value with the *pencil icon button* next to it's field as follows:
>     | enter your server address followed by `/pos_adyen/notification`

Reviewer: ```suggestion
#. click the **pencil icon** button and enter your server address, followed by
   `/pos_adyen/notification` in the :guilabel:`Event URLs` field;
```
